app/thirdparty/java/other/writeily/widget/WrFilesWidgetFactory.java

```diff
@@ -62,11 +63,11 @@ private void updateFiles() {
         } else if (dir.equals(GsFileBrowserListAdapter.VIRTUAL_STORAGE_FAVOURITE)) {
             _widgetFilesList.addAll(ApplicationObject.settings().getFavouriteFiles());
         } else if (dir.exists() && dir.canRead()) {
-            final File[] all = dir.listFiles(file -> true);
+            final boolean showDot = as.isFileBrowserFilterShowDotFiles();
+            final File[] all = dir.listFiles(file -> showDot || !file.getName().startsWith("."));
```

Comment:
Respect show dotfiles in widget as well

Reply:
This means the dot filter only applies to dotFiles in real, existing folders. But when you have i.e. a dotFile in the recents or favorites it is still shown.

Thats actually how I like it the most, but just noting that. This means i.e. a favorited `.gitignore` shows up.

Reply:
I agree. Though now that I am looking, is there actually a way to create widgets for favorites etc?

Reply:
yes if you know what you do :D. you can press up->up-> till you reach /storage, and favorites is ""virtual mounted"" at /storage/favorites for example.